chris_backend/plugins/serializers.py

```diff
@@ -132,7 +132,8 @@ def validate_version(self, version):
         if not isinstance(version, str):
             raise serializers.ValidationError(["Invalid type for plugin app version "
                                                "field. Must be a string."])
-        if not re.match(r"^[0-9.]+$", version):
+        re_valid_version = re.compile(r"^[0-9.]+$")
```

Comment:
Just as you said so yourself https://github.com/FNNDSC/ChRIS_ultron_backEnd/issues/383#issuecomment-1100901349

Reply:
I think it'd be fine "at the module level" (i.e. "As an outright global"). Maybe if you search the codebase for other occurrences of inline regex or otherwise inefficient patterns, you could create a module `constants.py` and import it from there.

Reply:
Okay this makes sense. I will do as you've suggested.